flows/vision-zero/cris_import/cris_import.py

```diff
@@ -40,31 +35,71 @@
-AWS_CSV_ARCHIVE_PATH_PRODUCTION = kv_dictionary["AWS_CSV_ARCHIVE_PATH_PRODUCTION"]
-AWS_CSV_ARCHIVE_PATH_STAGING = kv_dictionary["AWS_CSV_ARCHIVE_PATH_STAGING"]
-
-DB_HOST = kv_dictionary["DB_HOST"]
-DB_USER = kv_dictionary["DB_USER"]
-DB_PASS = kv_dictionary["DB_PASS"]
-DB_NAME = kv_dictionary["DB_NAME"]
-DB_IMPORT_SCHEMA = kv_dictionary["DB_IMPORT_SCHEMA"]
-
-DB_BASTION_HOST = kv_dictionary["DB_BASTION_HOST"]
-DB_RDS_HOST = kv_dictionary["DB_RDS_HOST"]
+SFTP_ENDPOINT = None
+ZIP_PASSWORD = None
+VZ_ETL_LOCATION = None
+
+AWS_DEFAULT_REGION = None
+AWS_ACCESS_KEY_ID = None
+AWS_SECRET_ACCESS_KEY = None
+AWS_CSV_ARCHIVE_BUCKET_NAME = None
+AWS_CSV_ARCHIVE_PATH_PRODUCTION = None
+AWS_CSV_ARCHIVE_PATH_STAGING = None
+
+DB_HOST = None
+DB_USER = None
+DB_PASS = None
+DB_NAME = None
+DB_SSL_REQUIREMENT = None
+
+DB_BASTION_HOST_SSH_USERNAME = None
+DB_BASTION_HOST = None
+DB_RDS_HOST = None
+
+if False:
+    kv_store = get_key_value("Vision Zero Development")
+    kv_dictionary = json.loads(kv_store)
+    SFTP_ENDPOINT = kv_dictionary["SFTP_ENDPOINT"]
+    ZIP_PASSWORD = kv_dictionary["ZIP_PASSWORD"]
+    VZ_ETL_LOCATION = kv_dictionary["VZ_ETL_LOCATION"]
+
+    AWS_DEFAULT_REGION = kv_dictionary["AWS_DEFAULT_REGION"]
+    AWS_ACCESS_KEY_ID = kv_dictionary["AWS_ACCESS_KEY_ID"]
+    AWS_SECRET_ACCESS_KEY = kv_dictionary["AWS_SECRET_ACCESS_KEY"]
+    AWS_CSV_ARCHIVE_BUCKET_NAME = kv_dictionary["AWS_CSV_ARCHIVE_BUCKET_NAME"]
+    AWS_CSV_ARCHIVE_PATH_PRODUCTION = kv_dictionary["AWS_CSV_ARCHIVE_PATH_PRODUCTION"]
+    AWS_CSV_ARCHIVE_PATH_STAGING = kv_dictionary["AWS_CSV_ARCHIVE_PATH_STAGING"]
+
+    DB_HOST = kv_dictionary["DB_HOST"]
+    DB_USER = kv_dictionary["DB_USER"]
+    DB_PASS = kv_dictionary["DB_PASS"]
+    DB_NAME = kv_dictionary["DB_NAME"]
+    DB_SSL_REQUIREMENT = kv_dictionary["DB_SSL_REQUIREMENT"]
+
+    DB_BASTION_HOST_SSH_USERNAME = kv_dictionary["DB_BASTION_HOST_SSH_USERNAME"]
+    DB_BASTION_HOST = kv_dictionary["DB_BASTION_HOST"]
+    DB_RDS_HOST = kv_dictionary["DB_RDS_HOST"]
+else:
+    SFTP_ENDPOINT = os.getenv("SFTP_ENDPOINT")
+    ZIP_PASSWORD = os.getenv("ZIP_PASSWORD")
+    VZ_ETL_LOCATION = os.getenv("VZ_ETL_LOCATION")
+
+    AWS_DEFAULT_REGION = os.getenv("AWS_DEFAULT_REGION")
+    AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+    AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+    AWS_CSV_ARCHIVE_BUCKET_NAME = os.getenv("AWS_CSV_ARCHIVE_BUCKET_NAME")
+    AWS_CSV_ARCHIVE_PATH_PRODUCTION = os.getenv("AWS_CSV_ARCHIVE_PATH_PRODUCTION")
+    AWS_CSV_ARCHIVE_PATH_STAGING = os.getenv("AWS_CSV_ARCHIVE_PATH_STAGING")
+
+    DB_HOST = os.getenv("DB_HOST")
+    DB_USER = os.getenv("DB_USER")
+    DB_PASS = os.getenv("DB_PASS")
+    DB_NAME = os.getenv("DB_NAME")
+    DB_SSL_REQUIREMENT = os.getenv("DB_SSL_REQUIREMENT")
+
+    DB_BASTION_HOST_SSH_USERNAME = os.getenv("DB_BASTION_HOST_SSH_USERNAME")
+    DB_BASTION_HOST = os.getenv("DB_BASTION_HOST")
+    DB_RDS_HOST = os.getenv("DB_RDS_HOST")
```

Comment:
Ugly, but useful for local testing. This is a thing I won't miss about the dockerized-agent. This could be more elegant.